pkg/terraform/stages/ovirt/stages.go

```diff
@@ -0,0 +1,16 @@
+package ovirt
+
+import (
+	"github.com/openshift/installer/pkg/terraform"
+	"github.com/openshift/installer/pkg/terraform/stages"
+	ovirttypes "github.com/openshift/installer/pkg/types/ovirt"
+)
+
+// PlatformStages are the stages to run to provision the infrastructure in oVirt.
+var PlatformStages = []terraform.Stage{
+	stages.NewStage(ovirttypes.Name, "image", stages.WithNormalDestroy()),
+	stages.NewStage(ovirttypes.Name, "cluster"),
+	stages.NewStage(ovirttypes.Name, "bootstrap", stages.WithNormalDestroy()),
+}
+
+// Note that the bootstrap IP and control-plane IP extraction is not supported on oVirt
```

Comment:
I cannot remember where this note was to indicate that I had not yet done the work to support IP extraction or whether IP extraction was not working even before these changes.